app/1.0/start/toolbox/set-up.md

```diff
@@ -16,6 +16,10 @@ App Toolbox template in less than 15 minutes.
 1.  Install the LTS version (4.x) of Node.js. The current version (6.x) should work, but is not
     officially supported. Versions below LTS are not supported.
 
+1. If you don't have them installed, install bower and gulp
+
+        npm install -g bower gulp
```

Comment:
@justinfagnani Do we currently make an attempt to resolve/install these dependencies or warn the user about them not being available or is this line worth adding to the setup docs for toolbox?